Client/core/CGUI.cpp

```diff
@@ -119,12 +129,8 @@ void CLocalGUI::ChangeLocale(const char* szName)
     if (guiWasLoaded)
     {
         CreateWindows(guiWasLoaded);
-
-        if (m_pConsole != nullptr)
-        {
-            m_pConsole->SetPosition(vPos);
-            m_pConsole->SetSize(vSize);
-        }
+        m_pConsole->SetPosition(consolePos);
+        m_pConsole->SetSize(consoleSize);
     }
```

Comment:
Is it possible that `m_pConsole` is `nullptr` here?

Reply:
If you call `DestroyWindows` prior then yes

Reply:
> Is it possible that `m_pConsole` is `nullptr` here?

No because it is created just before.